### PR TITLE
Improved Indonesian localization

### PR DESCRIPTION
**Pull request by januridp**
Improved Indonesian localization

### CODE REVIEW

File: language/phpmailer.lang-id.php [hunk 1]
```diff
@@ -5,22 +5,22 @@
...
-$PHPMAILER_LANG['mailer_not_supported'] = 'Mailer tidak didukung';
-$PHPMAILER_LANG['recipients_failed']    = 'Kesalahan SMTP: Alamat tujuan berikut menghasilkan error : ';
+$PHPMAILER_LANG['mailer_not_supported'] = 'Pengirim tidak didukung';
+$PHPMAILER_LANG['recipients_failed']    = 'Kesalahan SMTP: Alamat tujuan berikut menghasilkan kesalahan : ';
 $PHPMAILER_LANG['signing']              = 'Kesalahan dalam tanda tangan : ';
 $PHPMAILER_LANG['smtp_connect_failed']  = 'SMTP Connect() gagal.';
-$PHPMAILER_LANG['smtp_error']           = 'Kesalahan peladen SMTP : ';
+$PHPMAILER_LANG['smtp_error']           = 'Kesalahan pada pelayan SMTP : ';
 $PHPMAILER_LANG['variable_set']         = 'Tidak berhasil mengatur atau mengatur ulang variable : ';
 //$PHPMAILER_LANG['extension_missing']    = 'Extension missing: ';
```
Comment (Synchro) [inline language/phpmailer.lang-id.php]: Could you add this missing translation too?

Comment (januridp) [reply language/phpmailer.lang-id.php]: you mean extension_missing ?

Comment (Synchro) [reply language/phpmailer.lang-id.php]: Yes

Comment (januridp) [reply language/phpmailer.lang-id.php]: of course with pleasure

**Comment by Synchro** [comment]
Thank you very much!

**Comment by januridp** [comment]
you're welcome @Synchro

**Pull request merged**